types/postman-sandbox/index.d.ts

```diff
@@ -0,0 +1,52 @@
+// Type definitions for Postman sandbox.
+// Project: https://www.getpostman.com/docs/sandbox
+// Definitions by: Uroš Jarc <https://github.com/urosjarc/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * ENV
+ */
+declare const environment: Object;
```

Comment:
Don't use`Object` type here. better to use `object` . You will want to add `// TypeScript Version: 2.2` for the header as `object` is 2.2 feature